extensions/smallrye-reactive-messaging/deployment/src/main/java/io/quarkus/smallrye/reactivemessaging/deployment/DotNames.java

```diff
@@ -10,17 +10,19 @@
 import io.smallrye.reactive.messaging.annotations.Emitter;
 import io.smallrye.reactive.messaging.annotations.Merge;
 import io.smallrye.reactive.messaging.annotations.OnOverflow;
-import io.smallrye.reactive.messaging.annotations.Stream;
 
 public final class DotNames {
 
     static final DotName VOID = DotName.createSimple(void.class.getName());
     static final DotName INCOMING = DotName.createSimple(Incoming.class.getName());
     static final DotName OUTGOING = DotName.createSimple(Outgoing.class.getName());
-    static final DotName CHANNEL = DotName.createSimple(Stream.class.getName());
-    static final DotName STREAM = DotName.createSimple(Channel.class.getName());
-    static final DotName EMITTER = DotName.createSimple(Emitter.class.getName());
-    static final DotName ON_OVERFLOW = DotName.createSimple(OnOverflow.class.getName());
+    static final DotName CHANNEL = DotName.createSimple(org.eclipse.microprofile.reactive.messaging.Channel.class.getName());
+    static final DotName OLD_CHANNEL = DotName.createSimple(Channel.class.getName());
```

Comment:
Minor complaint: Maybe all this `old` stuff should be renamed to `deprecated`?

Reply:
I was wondering, "legacy" vs. "old" vs. "deprecated". I'm good with all of them. Any opinion?

Reply:
I like `legacy`! I didn't think of that one, excellent!

Reply:
Changed!

Reply:
Lovely! Now that the naming has been addressed, the hard part is over :laughing: